scipy/signal/signaltools.py

```diff
@@ -2220,20 +2220,20 @@ def resample(x, num, t=None, axis=0, window=None):
     N = int(np.minimum(num, Nx))
     Y = zeros(newshape, 'D')
     sl[axis] = slice(0, (N + 1) // 2)
-    Y[sl] = X[sl]
+    Y[tuple(sl)] = X[tuple(sl)]
     sl[axis] = slice(-(N - 1) // 2, None)
-    Y[sl] = X[sl]
+    Y[tuple(sl)] = X[tuple(sl)]
```

Comment:
Probably faster to not call tuple twice here

Reply:
I think for the code that is executed here the speed difference will be negligible, and I'm not sure it would help readability to assign it to a temp var.

Reply:
Also I suppose numpy was calling `tuple` twice before anyway internally, so this is not really any slower.